src/celeritas/field/FieldDriver.hh

```diff
@@ -304,7 +303,7 @@ FieldDriver<StepperT>::integrate_step(real_type       step,
         // Compute a proposed new step
         CELER_ASSERT(output.end.step > 0);
         output.proposed_step = this->new_step_size(
-            step, dyerr / (step * options_.epsilon_step));
+            step, dyerr / (options_.epsilon_step * step));
```

Comment:
Do we (and/or NVCC) care about the order of operation or it this stylistic?

Reply:
Stylistic; just for consistency with the other uses in the file